packages/server/src/portal-sync/portal-sync.service.ts

```diff
@@ -106,23 +106,49 @@ type SocrataDataset = {
  * suit your testing needs.
  */
 const DATA_REFRESH_CONFIG = {
-  // set this to true to ignore populating elastic search
-  skipElasticSearchRebuild: false,
+  // Set this to true to ignore populating elastic search. There is no env var
+  // for this. This is useful when testing data ingestion.
+  skipElasticSearchRefresh: false,
+
+  // Set this to true to avoid refreshing postgres. There is no env var for
+  // this. This is useful when testing data ingestion.
   skipPostgresRefresh: false,
+
+  // Set this to true to avoid processing the dataset descriptions with
+  // tensorflow. This is useful when testing data ingestion, since the ML step
+  // is the slowest. There is no env var for this.
   skipMLDatasetProcessing: false,
 
-  // set this to true to use a hardcoded list of portals
+  // Set this to true if you want to recreate the elasticsearch index from
+  // scratch. It's recommended to pass this as an env var instead.
+  // Env var: RECREATE_ELASTICSEARCH_INDEX
+  recreateElasticsearchIndex: false,
+
+  // Set this to true to use a hardcoded list of portals. If a
+  // PORTAL_OVERRIDE_LIST env var is set, then we will consider
+  // `usePortalListOverride` to be true even if this isn't explicitly set.
   usePortalListOverride: false,
 
+  // This is used to control what datasets get updated in Elasticsearch.
+  // Only datasets with a `metadataUpdatedAt` after this date will be updated.
+  // Indexing to Elasticsearch is expensive so we try to avoid updating *all*
+  // datasets. Set this timestamp to an old date (e.g. '1970-01-01') if you want
+  // to update all datasets. Dates must be a YYYY-MM-DD string.
+  // It's recommended to pass this as an env var instead.
+  // Env var: LAST_METADATA_UPDATE_DATE
+  lastMetadataUpdateDate: '2022-01-01',
```

Comment:
would this need to be manually updated later?

Reply:
This would be overridden via a environment variable when we run the data ingestion command (`yarn seed-database-full`). The cron job we set will set this environment variable correctly whenever we run this on a weekly basis